perma_web/perma/migrations/0017_rename_mylinks_to_personallinks.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.core.exceptions import ObjectDoesNotExist
+
+
+def rename_folders_helper(apps, schema_editor, old, new):
+    Folder = apps.get_model("perma", "Folder")
+    try:
+        folders = Folder.objects.filter(name=old)
```

Comment:
Suggest adding the filter `is_root_folder=True`, to avoid changing any folders that happen to be called "My Links".